manager/probe.go

```diff
@@ -63,16 +63,17 @@ func (pip ProbeIdentificationPair) Matches(id ProbeIdentificationPair) bool {
 // Probe - Main eBPF probe wrapper. This structure is used to store the required data to attach a loaded eBPF
 // program to its hook point.
 type Probe struct {
-	manager          *Manager
-	program          *ebpf.Program
-	programSpec      *ebpf.ProgramSpec
-	perfEventFD      *internal.FD
-	state            state
-	stateLock        sync.RWMutex
-	manualLoadNeeded bool
-	checkPin         bool
-	funcName         string
-	attachPID        int
+	manager           *Manager
+	program           *ebpf.Program
+	programSpec       *ebpf.ProgramSpec
+	perfEventFD       *internal.FD
+	state             state
+	stateLock         sync.RWMutex
+	manualLoadNeeded  bool
+	checkPin          bool
+	funcName          string
+	attachPID         int
+	attachedWithSysfs bool
```

Comment:
Could we call this `attachedWithDebugFS`? Since the other method does accesses files in `/sys/bus/event_source/devices/`, this name could be confusing.

Reply:
👍🏻